historical_system_profiles/models.py

```diff
@@ -51,14 +59,6 @@ def _get_utc_aware_dt(self, datetime_in):
     def display_name(self):
         return self.system_profile["display_name"]
 
-    @property
```

Comment:
I would leave this here for the time being since the view still calls it. We can take it out later once we are sure that `captured_on` is always populated